.github/workflows/tests.yml

```
@@ -118,6 +118,11 @@ jobs:
           name: gmt-cache
           path: .gmt
 
+      # Pull baseline image data from dvc remote (DAGsHub)
+      - name: Pull baseline image data from dvc remote
+        run: |
+          dvc pull
```

```suggestion
        run: dvc pull
```

This can be on one line, unless you want to have `ls -lhR test/api test/baseline ...` lines too.